example_byteslice_bigendian_test.go

```diff
@@ -9,3 +9,11 @@ func ExampleLSet() {
 	fmt.Printf("%x\n", LSet(data, setData))
 	// Output: bada55
 }
+
+func ExampleLUnset() {
+	data := []byte{0xAA, 0xCA, 0x55}
+	setData := []byte{0x10, 0x12}
+
+	fmt.Printf("%x\n", LUnset(data, setData))
+	// Output: 000255
```

Comment:
The output is not bada55, please change the data and the setData to produce this output